tools/releaser/main.go

```diff
@@ -220,11 +244,17 @@ This utility is intended to handle many of the steps to release a new version.
 	return nil
 }
 
-func copyHelper(destPath, srcPath string) error {
+func copyFile(destPath, srcPath string) error {
```

Comment:
```suggestion
func copyFile(workspace, destPath, srcPath string) error {
...
   path.Join(Dir...)
```

Reply:
I'd go the other way. Change the signature to `copyFile(path string) error` and then construct `src := path.Join(os.Getenv("BUILD_WORKSPACE_DIRECTORY"), "bazel-bin", path)` and `dest := path.Join(os.Getenv("BUILD_WORKSPACE_DIRECTORY"), path)`. No reason to overly generalize a bespoke helper.